### PR TITLE
feat: iOS - background mode, delay start of audio

### PR DESCRIPTION
**Pull request by webdog-rich**
Added support for background mode on iOS - must be enabled in iOS project capabilities.
Added option to delay start of audio - iOS only (for now!)

### CODE REVIEW

**Comment by lordrepha1980** [comment]
please merge it in main. I need this.

**Comment by riderx** [comment]
Thanks a lot !
i think you need to run `npm run fmt` and then i will be able to accept it !

**Comment by webdog-rich** [comment]
ok, I have run and pushed.

File: ios/Plugin/Plugin.swift [hunk 1]
```diff
@@ -35,15 +33,73 @@ public class NativeAudio: CAPPlugin {
...
+
+            }
+
+        }
+
+        if let background = call.getBool(Constant.Background) {
+
+            do {
+
+                if background {
```
Comment (riderx) [inline ios/Plugin/Plugin.swift:59]: I’m not sure why the if is repeated here with line 56.

i think you can give a default value to the getter and then remove the if assign and the double scope

**Comment by riderx** [comment]
Thanks a lot ,can you check your conflict as well ?

**Comment by lordrepha1980** [comment]
What happens with that now?

**Comment by lordrepha1980** [comment]
I don't know where the problem is right now, but I really need this feature urgently. Is there a status on whether it works in the current release or not?

**Comment by riderx** [comment]
Hey @lordrepha1980 I think this can be handled by a community member if you are willing to support it.
Usually between 30 to $50 would make someone want to make it happen in 3/4 days.

**Comment by webdog-rich** [comment]
Sorry, I've been busy on other projects. Conflict should be resolved!

**Pull request merged**